mm/kfence/kfence_core.c

```diff
@@ -50,6 +50,9 @@ static unsigned long kfence_pool_start, kfence_pool_end;
 /* Protects kfence_freelist, kfence_recycle, kfence_metadata */
 static DEFINE_SPINLOCK(kfence_alloc_lock);
 
+#define KFENCE_DUMP_BUF_SIZE (PAGE_SIZE * 2)
```

Comment:
Since this is private to this compilation unit, why not save us a macro and just have:

/* Size is chosen to accommodate ...?? */
static char kfence_dump_buf[PAGE_SIZE * 2];

and below just use sizeof(kfence_dump_buf).

Reply:
Yes, good idea.